RELEASE_PROCESS.md

```diff
@@ -20,92 +40,278 @@ Regular releases concern only major and minor releases. We do not decide up fron
+
+Every feature added to the [release branch](#step-3---create-release-branches) needs to be properly described in the release notes post.
+
+
+### Step 10 - prepare release notes
+
+In addition to the [announcement blog post](#step-9---update-announcement-blog-post), the [release coordinator](#%22release-coordinator%22) should prepare release notes for each of the [repositories covered by this process](#repositories).
 
-There are no step-by-step instructions to follow but a set of rules.
+These should:
+- be written in markdown
+- include a list of features and significant changes
+- include a link to github issues and pull requests where more background can be found
+- include a link to a github diff with the previous release
 
-#### Review
+Examples of doing this are:
+- the [release notes for version 2.3.0 of spec](https://github.com/asyncapi/spec/releases/tag/v2.3.0)
 
-- Everybody is invited to perform a review of proposals. The [contribution guide](CONTRIBUTING.md) describes all requirements proposal needs to fulfill to reach `Stage 3: Accepted`,
-- Pull request must be:
-  - labeled as an accepted proposal,
-  - created against the feature branch,
-  - created in all repositories specified in contribution guide,
-- At least one user listed in [CODEOWNERS](CODEOWNERS) must approve the pull request in all related repositories.
 
-#### Merge
+### Step 11 - notify people the release is coming
 
-- Merge can be done only by repository [CODEOWNERS](CODEOWNERS),
-- Every pull request must have a `feat: ` prefix that, after the merge, produces a release candidate with a minor version update. The major version must have `feat!: ` prefix,
-- Pull request in the `parser-js` can be merged only if it uses the release candidate of `@asyncapi/specs` package produced after merge of a pull request in `spec-json-schemas`,
-- First, changes are merged into the `spec` repository, then `spec-json-schemas` and then in `parser-js`.
+As the release gets closer to being ready, it is helpful to remind the community that the release is on the way.
 
-### Next steps
+This can be done:
+- in [Slack](https://www.asyncapi.com/slack-invite)
+- at [community meetings](https://github.com/asyncapi/community/issues?q=is%3Aissue+is%3Aopen+label%3Ameeting)
 
-- Update previously created draft of release notes with information about the new feature. Collaborate closely with feature contributors,
-- Notify maintainers of the following repositories that the first feature is merged and that release will be produced and therefore they need to start preparing for it:
+Including a link to the [release issue](#step-2---create-a-release-issue) is a good way to let the community see the progress that has been made so far.
+
+
+### Step 12 - reviews
+
+At least one [code owner](#code-owners) must approve the [release pull requests](#step-7---create-draft-pull-requests) in all related [repositories](#repositories).
+
+
+### Step 13 - release candidates
+
+_how is this done?_
```

Comment:
@derberg Can you help fill me in on how this works, please?

Reply:
Once PR with fix or feat prefix is merged into release branch, automation and bot produces a prerelease, release candidate that includes in name upcoming release number, date of planned release and also the number of the candidate. Example https://github.com/asyncapi/spec/releases/tag/v2.3.0-2022-01-release.3

Important to know is that if `@asyncapi/specs` candidate is released, then PP in Parser-js for the same feature should be updated to use the release candidate, so the released release candidate for parser actually contains related changes in JSON Schema